### ISSUE TITLE
TimestampType

### ISSUE THREAD

**Issue by zhiboz** [opencypher/cypher-for-apache-spark#229]
It appears the following data types, and arrays of these types, are supported

- StringType 
- LongType 
- BooleanType
-  BinaryType
 - DoubleType

It'd be nice for this package to support `TimestampType`. 

Thanks a lot!

**Comment by zhiboz** [comment]
The following error occurs when trying to import `timestamp` field `from_date`

```
Exception in thread "main" org.opencypher.caps.api.exception.CAPSException: Cannot (yet) handle the given argument Cannot convert or cast type TimestampType of field StructField(from_date,TimestampType,true) to a Spark type supported by Cypher
```

**Comment by Mats-SX** [comment]
Hi @zhiboz and thanks for reaching out to us!

The goal of this project is firstly to support all parts of the Cypher language. Unfortunately, Cypher has not yet fully adopted a specification for datetime support, although [a proposal has been in existence for some time](https://github.com/opencypher/openCypher/pull/15). Once the oCIG has accepted this (or some other) proposal, I would expect us to plan to add these capabilities also to CAPS.

**Comment by Mats-SX** [comment]
This is solved since #743
